File: Python Exams/Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/02. Cat Walking/02. Cat Walking.py
minutes =  int(input())
day_walk = int(input())
calories = int(input())

all_walk = minutes * day_walk
burnet_calories = all_walk  * 5

if calories / 2 <= burnet_calories:
    print(f'Yes, the walk for your cat is enough. Burned calories per day: {burnet_calories}.')
else:    
    print(f'No, the walk for your cat is not enough. Burned calories per day: {burnet_calories}.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/02. Cat Walking/02. Cat Walking.js
function catWalking(minutes, dayWalk, calories) {
    let allWalk = minutes * dayWalk
    let burnetCalories = allWalk * 5

    if (calories / 2 <= burnetCalories)
        return `Yes, the walk for your cat is enough. Burned calories per day: ${burnetCalories}.`
    return `No, the walk for your cat is not enough. Burned calories per day: ${burnetCalories}.`
}

//console.log(catWalking(30, 3, 600))